### ISSUE TITLE
Won't allow me to add WebRTC.framework

### ISSUE THREAD

**Issue by MasamMahmood** [tkmn0/SimpleWebRTCExample_iOS#12]
I have no idea but I am following your way but it wont allow me to add WebRTC.framwork. Obviously its the simple thing but no success. 

using swift 5 & Xcode 11.3 (recently updated to)  Mac OS Catalina

**Comment by tkmn0** [comment]
Hi, thank you for your question.

I will check my way ( [how to add]( https://github.com/tkmn0/SimpleWebRTCExample_iOS/blob/master/docs/how_to_add.md) )  with your situation.

**Comment by MasamMahmood** [comment]
Hi sure and thank you.

**Comment by MasamMahmood** [comment]
@tkmn0  Any update regarding the issue ?

**Comment by tkmn0** [comment]
Hi, Sorry for replying so late.

I tried clone this repository, install frameworks with pod and drag and drop WebRTC.framework to my project follow my way.
Then, Xcode said `no such module WebRTC`. 
In my case, I deleted WebRTC.framework from my project once, then, I paste WebRTC.framework to SimpleWebRTCExample_iOS folder firstly,  finally, I drag & drop that WebRTC.framework into Xcode Project.
<img width="728" alt="スクリーンショット 2019-12-16 23 32 07" src="https://user-images.githubusercontent.com/22463862/70915205-c99fe200-205c-11ea-9960-d222d6df98af.png">


Feel free to ask me any questions with more detail information :)
